test/testFoo.cpp

```diff
@@ -1,8 +1,30 @@
+#include "MPM.hpp"
 #include "mesh.hpp"
 #include "materialPoints.hpp"
-#include "integration.hpp"
+#include "assembly.hpp"
+
+#include "testUtils.hpp"
 
 int main() {
-  auto m = Mesh();
-  return 0;
+
+    //test init
+    auto m = polyMpmTest::Mesh();
+    auto mp = polyMpmTest::MaterialPoints();
+    auto v = polyMpmTest::Vector2();
+    //auto mpm = polyMpmTest::MPM();
+    
+    //test init Test Mesh
+    Kokkos::initialize();{
```

Comment:
`Kokkos::initialize();` should be one of the first functions called in `main()`

Reply:
https://github.com/SCOREC/polyMpmTest/commit/e417849f4ca97cd31cb59d812e0e90920c930a28#diff-b644603964e68f7440b21ec7c2eee992850906dd722495525d95f61e784dd134R10